kamon-play-2.5.x/src/main/scala/kamon.play/instrumentation/mixin/InjectTraceContext.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * =========================================================================================
+ * Copyright © 2013-2017 the kamon project <http://kamon.io/>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ * =========================================================================================
+ */
+
+package kamon.play.instrumentation.mixin
+
+import kamon.trace.{TraceContext, TraceContextAware, Tracer}
+
+/**
+  * Mixin for play.api.mvc.RequestHeader
+  */
+class InjectTraceContext extends TraceContextAware {
+  // This has be a lazy val
+  @transient lazy val traceContext: TraceContext = Tracer.currentContext
```

Comment:
I have no idea how that got there, but it's not a bad idea turn off the serialization for that field.